src/app/parent-component/parent.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'parent',
+    templateUrl: './parent.component.html',
+    styleUrls: ['./parent.component.css']
+})
+
+export class ParentComponent {
+
+    childTitle: string = 'text passed to child';
+    
+    constructor(){}
+
+    onNotify() {
```

Comment:
capture the data passed from `parentNotified` into `onNotify` method and display that here